.gitignore

```diff
@@ -1,2 +1,32 @@
 *.exe
 .vs/
+
```

Comment:
While a good idea, this addresses a symptom and not the cause. Instead you should add a clean target to the Makefile and get autotools to clean up after itself. Doing those two things will make it so the amount of files is minimal/easier to clean.